feat(docs): tls document improvement (#373)
emqx/cloud-docs: en_US/deployments/port_guide_dedicated.md

# Dedicated connection guide

## Ports overview

| **Plans**  | **TCP port(mqtt)**   | **MQTT over TLS/SSL port (mqtts)**   |**WebSocket port(ws)**   | **WebSocket over TLS/SSL port(wss)**   |
| -------------- | ----------------------- | ------------------|------------------|------------------|
| Standard     | 15xxx      | 15xxx         |8083      | 8084         |
| Professional     | 1883     | 8883 (Need TLS/SSL configuration)  | 8083    | 8084 (Need TLS/SSL configuration)       |

### What are MQTT over TCP and WebSocket?

> The standard MQTT is an asynchronous communication messaging protocol built on the TCP/IP stack and is a lightweight publish and subscribe messaging protocol. It can scale in unreliable network environments and is suitable for scenarios where device hardware storage space or network bandwidth is limited. webSocket protocol is a new network protocol based on TCP, where the browser and server only need to complete one handshake, and a persistent connection can be created directly between the two and bi-directional data transfer, and data exchange between the client and server becomes more The exchange of data between client and server becomes much simpler. Both standard MQTT and WebSocket are unencrypted.

Reviewer: ```suggestion
The standard MQTT is a lightweight publish and subscribe messaging protocol for asynchronous data communication. It is built on the TCP/IP stack and can scale in unreliable network environments. Therefore, it is suitable for scenarios where device hardware storage space or network bandwidth is limited. WebSocket protocol is a new network protocol based on TCP, which enables the creation of a persistent connection between a browser and a server through a single handshake. Since there is no need for repeated handshakes between the browser and server, bi-directional data exchange between the two becomes much simpler. Both standard MQTT over TCP and WebSocket are unencrypted, so there can be safety risks.
```

> The MQTT over TCP port is 1883, which corresponds to the 'mqtt', and the WebSocket port is 8083, which corresponds to the 'ws'.

Reviewer: ```suggestion
The MQTT over TCP port is 1883, which corresponds to the 'mqtt'. The WebSocket port is 8083, which corresponds to the 'ws'.
```

### How do I use the MQTT over TCP port and the WebSocket port?

You can view the sample code for testing and connecting by using SDK Demos at [Client Connection Guidelines](../connect_to_deployments/overview.md).

### What's MQTT over TLS/SSL and WebSocket over TLS/SSL？

Add TLS/SSL encryption to MQTT or WebSocket protocol communication to ensure that the communication is secure from eavesdropping and tampering.

> The MQTT over TLS/SSL port is 8883, which corresponds to the 'mqtts', and the WebSocket over TLS/SSL port is 8084, which corresponds to the 'wss'.

Reviewer: ```suggestion
The MQTT over TLS/SSL port is 8883, which corresponds to the 'mqtts'. The WebSocket over TLS/SSL port is 8084, which corresponds to the 'wss'.
```
